akka-persistence-typed-tests/src/test/scala/akka/persistence/typed/scaladsl/EventSourcedBehaviorRetentionSpec.scala

```diff
@@ -127,6 +127,41 @@ class EventSourcedBehaviorRetentionSpec
   val pidCounter = new AtomicInteger(0)
   private def nextPid(): PersistenceId = PersistenceId.ofUniqueId(s"c${pidCounter.incrementAndGet()}")
 
+  "EventSourcedBehavior with predicate" must {
```

Comment:
A bit vague perhaps, what about:
```suggestion
  "EventSourcedBehavior with snapshot predicate" must {
```